src/ztc/api/serializers/resultaattype.py

```diff
@@ -6,6 +6,9 @@
     GegevensGroepSerializer, NestedGegevensGroepMixin,
     add_choice_values_help_text
 )
+from vng_api_common.validators import ResourceValidator
+
+from ztc.conf.api import REFERENTIELIJSTEN_API_SPEC
```

Comment:
Never import settings directly from the settings file, you should always use `from django.conf import settings` and access the setting from the `settings` object